src/main/kotlin/day05/Encapsulation.kt

```
@@ -0,0 +1,29 @@
+package day05
+
+import day03.Hero
+
+fun main() {
+
+}
+
+class Encapsulation {
+}
```

detekt (1.23.7)

[warning] The class or object Encapsulation is empty.

(detekt.empty-blocks.EmptyClassBlock)